addons/languages/pilaga/apk/build.gradle

```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```

Comment:
For the first PR, we'll need this to be set to `false`. After a successful manual release, we can switch to `true`

Reply:
Done. Thanks!